flashinfer/gemm/gemm_base.py

```
@@ -3755,3 +3794,409 @@ def fp8_blockscale_gemm_sm90(
+    a_shape,
+    a_stride,
+    a_type,  # cudnn.data_type, FP8_E4M3 or FP8_E5M2
+    b_shape,
+    b_stride,
+    b_type,  # cudnn.data_type, FP8_E4M3 or FP8_E5M2
+    block_size,
+    o_type,  # cudnn.data_type, BF16 or FP16
+    device,
+):
+    if len(a_shape) != 3:
+        raise ValueError(f"A shape must be 3D, got {a_shape}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)